Allowing behavior: hide on checkbox checked

jersiovic opened the pull request:
Solves this issue https://github.com/OrchardCMS/Orchard2/issues/518

Now a checkbox can be set to collapse sth when it is checked setting its attribute `data-toggle="collapse in"`

So, in the sample mentioned in the issue, the desired behaviour is accomplished in this way:
`<input asp-for="TestingModeEnabled" type="checkbox" data-toggle="collapse in" data-target="#certArea" class="form-check-input" checked="@Model.TestingModeEnabled"/>`

sebastienros commented:
Note that in the latest alpha for which @agriffard has a PR open, the class `in` doesn't exist anymore.

jersiovic commented:
So, best is I wait that PR is merged to see how things work with new Bootstrap upgrade.
I close this PR by the moment

jersiovic commented:
I've rebased this branch on master to get last bootstrap bits. 
Also I have replaced 'in' class per 'active' class in the patch.
Finally I force the update of collapse' and 'show' classes on the load of the page to avoid current Bootstrap problems when a checkbox is loaded.
With this fix and some cleaning in the code of OpenId views I get fixed all the problems I had with checkboxes that hide other content.
